homeassistant/components/frontend/__init__.py

```diff
@@ -357,8 +357,10 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     )
 
     local = hass.config.path("www")
-    if os.path.isdir(local):
-        hass.http.register_static_path("/local", local, not is_dev)
+    if not os.path.isdir(local):
+        os.mkdir(local)
```

Comment:
I kinda agree, but let's only create this once on initial set up / onboard (as part of the default configuration).

Reply:
This doing I/O and needs to be in executor.